README.md

```diff
@@ -1,6 +1,6 @@
 # alpinejs-devtools
 
-<p align="center"><img width="720px" src="alpine-devtools-screenshot.png" alt="screenshot"></p>
+<p align="center"><img width="720px" src="docs/alpine-devtools-screenshot.png" alt="screenshot"></p>
```

Comment:
total nitpick but any reason we're not using markdown's `![]()` for this? @Te7a-Houdini

Reply:
@HugoDF no reason we can use markdown for sure.

Reply:
> total nitpick but any reason we're not using markdown's `![]()` for this? @Te7a-Houdini

I did think this too, but kept it close to original. Will change now.

Reply:
> total nitpick but any reason we're not using markdown's `![]()` for this? @Te7a-Houdini

Doesn't look like GitHub understands the Markdown equivalent inside of a `<p>` tag, will leave as is.